crates/runtime/benches/bench.rs

```diff
@@ -187,17 +187,16 @@ async fn run_query_and_record_result(
     let mut min_iter_duration_ms = i64::MAX;
     let mut max_iter_duration_ms = i64::MIN;
 
+    let mut query_err: Option<String> = None;
+
+    let mut completed_iterations = 0;
+
     for _ in 0..benchmark_results.iterations() {
+        completed_iterations += 1;
```

Comment:
It would be better to put `completed_iterations += 1` should be put after `if let Err(e) = res {
            query_err = Some(e);
            break;
        }`. If a query failed in the first iteration and the loop breaks, we probably don't want to mark it to have 1 completed iteration

Reply:
Discussed with @Sevenannn / will keep existing implementation as failed query is actually a completed attempt/test.